docs/articles/licensing/gplv3/gplv3-blacklist.rst

```diff
@@ -0,0 +1,86 @@
+GPLv3 licensed production software Vs automotive industry
+---------------------------------------------------------
+
+Modern cars have around 100 million lines of code running on them,12 with 70% 
+of that code being in the head unit. Complexity is a non-trivial issue in 
+automotive software design. In addition to being complex, cars can be dangerous.
+
+Locking the software to the hardware by signing the original software image
+with a cryptographic key so that only an image provided by the supplier will 
+boot or install – is a common practice in embedded devices.
+This process of signing software images – so only images with the right key 
+will boot or install effectively prevents a user from modifying the software
+on the device since they have no access to the key needed to allow their 
+modified version to boot or install.
+
+With regard to the GPLv3, the difficulty is with only the anti-tivoization 
+clause, and the reason for disfavoring that license is safety. 
+
+References
+^^^^^^^^^^
+
+`<https://www.gnu.org/licenses/gpl-faq.en.html#Tivoization>`_
+`<http://www.ifosslr.org/ifosslr/article/view/102/207>`_
+
+Difference between GPLv3 license usage of production and development images
+---------------------------------------------------------------------------
+
+* The development image can contain GPLv3 softwares even without commercial
+  license. E.g - gdbserver(GPLv3) can be used in development image 
+  but not in production image.
+* In production image if GPLv3 non-commercial licensed software is used,
+  then the vendor has to provide the necessary tools to change the 
+  software as well.
+
+
+How to facilitate blacklisting of GPLv3 licensed software in Yocto
+------------------------------------------------------------------
+
+1. Add layer **meta-gplv2**
+2. Remove "initramfs-live-install" and "initramfs-module-install" from 
+   environment variable PACKAGE_INSTALL.
+   *(file: core-image-minimal-initramfs.bbappend)*
+
+   .. code-block:: bash
+
+      PACKAGE_INSTALL_remove = "initramfs-live-install initramfs-module-install"
+3. Remove "grub-efi" from WKS_FILE_DEPENDS_BOOTLOADERS for x86-64 arch.
+   *(file: image_types_wic.bbclass)*
+
+   .. code-block:: bash
+
+     WKS_FILE_DEPENDS_BOOTLOADERS_remove_x86-64 = "grub-efi"
+4. Remove "parted" from RDEPENDS_${PN}. 
+   *(file: initramfs-module-install-efi_1.0.bb)*
+
+   .. code-block:: bash
+
+     RDEPENDS_${PN}_remove = "parted"
+5. Select "system-boot" as the bootloader instead of default "grub-efi".
+   *(file: systemd-boot.bbclass)*
+
+   .. code-block:: bash
+
+     EFI_PROVIDER = "systemd-boot"
+6. Deprecate GPLv3 and other unwanted licenses as follows - 
+
+   .. code-block:: bash
+
+	INCOMPATIBLE_LICENSE = "\
+	    GPL-3.0 \
+	    LGPL-3.0 \
+	    AGPL-3.0 \
+	    MPL-1.1 \
+	    SGI-1 \
+	    CECILL-2.0 \
+	    CDDL-1.1 \
+	    EPL-1.0 \
+	    CPL-1.0 \
+	"
```

Comment:
I have no clue about half of these licenses and I believe that we should have proper comments why they are here? What is the background why wasn't only L/A/GPLv3 removed?

Reply:
Many of the licenses, if not all, are considered 'problematic' from a OEM's point of view, just not for the same reasons as the GPLv3. One option might be to describe only removing GPLv3 in this section and not focus on other licenses in this part of the SWF.

Reply:
Thanks @jeremiah ! That's good to know, I believe we should not address them in this chapter, because, like you have mentioned, there are other reasons not to include those.

Reply:
Can we refer to some other places that discuss bad/good licenses and their implications?